src/problems/vrptw/operators/pd_shift.cpp

```diff
@@ -183,9 +93,6 @@ void PDShift::apply() {
   if (_s_d_rank == _s_p_rank + 1) {
     _tw_s_route.remove(_input, _s_p_rank, 2);
   } else {
-    std::vector<Index> source_without_pd(s_route.begin() + _s_p_rank + 1,
-                                         s_route.begin() + _s_d_rank);
-
```

Comment:
Good find, wonder why this did not show up as a compiler warning before since the variable is not used...

Reply:
I guess it's because the compiler cannot determine that the constructor of std::vector doesn't do anything useful

Reply:
Right, makes sense.